index.html

```diff
@@ -81,35 +78,46 @@ <h1>I'm Adeyemi Glad to see you!</h1>
             </p>
           </div>
           <div class="Social-media">
-            <div class="socialText"> 
+            <div class="media-Container"> 
               <div class="socioText">
                 <p>LET'S CONNECT</p>
               </div>
             </div>
             <div class="social-media-icon">
-              <li>
-                <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a>
-                <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin icon.png" alt="Linkedin"></a>
-                <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
-                <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
-                <a class="handwe" href="#github.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
-              </li>
+              <ul>
+                <li>
+                  <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a>
+                </li>                
+                <li>
+                  <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin_icon.png" 
+                      alt="Linkedin"></a>
+                </li>
+                <li>
+                  <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
+                </li>
+                <li>
+                  <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
+                </li>
+                <li>
+                  <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
+                </li>  
+              </ul>            
             </div>
-            <div>
-                
+            <div> 
             </div>
           </div>
         </section>
       </main>
       <script>
-        function myFunction() {
-          var x = document.getElementById("icons-icons-menu");
+        function menuBar() {
+          var x = document.getElementById("myLink");
           if (x.style.display === "block") {
             x.style.display = "none";
           } else {
             x.style.display = "block";
           }
         }
         </script>
```

Comment:
**OPTIONAL** - Please keep all Javascript code out of your HTML page and leave them in a separate JS file. You can link the JS file to your HTML page. Having JS in your HTML will make your page load slower. Please fix this 👍 👍